app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="flex flex-col w-full items-center">
+<h2>Sign up</h2>
```

Comment:
- [Optional] I think this could have been a button/link for registering after filling in the input fields below. I could easily figure it out by clicking the enter key but other random users may not easily do the same which can lead to a bad user experience while using your app. Let's kindly fix this.